Gemfile

```diff
@@ -1,13 +0,0 @@
-source "https://rubygems.org"
-gem "nokogiri", ">= 1.10.4"
```

Comment:
Please do not remove those Gemfiles.

Reply:
They are ignored, left? https://github.com/galaxyproject/training-material/blob/master/.gitignore#L6